src/main/java/com/sarahhand/json/JsonReaderWriter.java

```diff
@@ -61,4 +63,35 @@ public JsonReaderWriter(PropertyNamingStrategy pns){
 			log.error("Error writing JSON to stream.", e);
 		}
 	}
-}
+
+	/** This creates an object out using a JSON file.
```

Comment:
Hmmmm . . . this isn't what I was thinking we'd have.  Does your JSON file contain a FractalEnvelope or a FractalConfig? I was thinking it would contain a FractalEnvelope, but this code makes me think it contains a FractalConfig.  Let's talk about this some more.  Let's look at how to clean this up so that this class has no references to fractal-specific classes - all that should be in ConfigSaveLoader.